pages/common/xml-transform.md

```diff
@@ -0,0 +1,12 @@
+# xml transform
+
+> Transform XML documents using XSLT.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Transform an XML document using an XSL stylesheet, passing one XPATH parameter and one literal string parameter:
+
+`xml transform {{path/to/stylesheet.xsl}} -p {{"Count='count(/xml/table/rec)'"}} -s {{Text="Count="}} {{path/to/input.xml|URI}}`
```

Comment:
What part of those arguments will actually be changed? Because we should only highlight those parts. For example for `Text` I think it should be `Text="{{Count=}}"`, but I'm not sure.

Reply:
This presumes a style sheet with these parameters (in their manual). Really hard to do this one as TLDR.

Reply:
```
{{Text="Count="}}
```
is correct as quoted.